CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
 
+## 0.11.2 [#61](https://github.com/openfisca/openfisca-ops/pull/61)
+
+* Use environement variables to configure the legislation explorer
+
+## 0.11.0 [#59](https://github.com/openfisca/openfisca-ops/pull/59)
+
+* Serve `fr.openfisca.org/api/v22`
+* Details:
+  - On repo:
+      - Created configuration files in `openfisca-ops/fr.openfisca.org/api/v22`
```

Comment:
> In general, documenting in a Changelog the creation/edition of a file is quite redundant, as `git` tools already provide this information.

Reply:
Agreed, I think we should focus on changes made to the server (new directories ...)